frontend/src/pages/dashboard/jios/list/allJios/JioCardList.tsx

```diff
@@ -44,14 +44,14 @@ export default function JioCardList() {
       if (jioSearchValues) {
```

Comment:
Perhaps it is better to be more explicit: jioSearchValues !== null ? !== undefined, etc